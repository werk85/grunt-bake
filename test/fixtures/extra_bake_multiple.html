<!DOCTYPE html>
<html>
	<head></head>
	<body>
	<!--(bake-start _foreach="group:groups" _bake="extra_entry.html > extra-{{group@index}}-{{group.name | lowercase}}.html")-->
		<a href="{{@link}}">Details for {{group.name}}</a>
	<!--(bake-end)-->
	</body>
</html>