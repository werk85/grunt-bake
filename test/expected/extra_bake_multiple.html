<!DOCTYPE html>
<html>
	<head></head>
	<body>
		<a href="extra-0-a-team.html">Details for A-Team</a>
		<a href="extra-1-b-team.html">Details for B-Team</a>
	</body>
</html>